robot_descriptions/loaders/idyntree.py

```diff
@@ -20,16 +20,24 @@
 """
 
 from importlib import import_module  # type: ignore
+from typing import List, Optional
+from .pinocchio import get_package_dirs
```

Comment:
We should move this function to a common file, now that it is not specific to the Pinocchio loader any more.

I'd say we move it to ``.._package_dirs``. What do you think?

Reply:
Moved in https://github.com/robot-descriptions/robot_descriptions.py/pull/24/commits/ef2db9c339df54cb58c9287d4035b8e569c0d22e